index.html

```diff
@@ -0,0 +1,490 @@
+                                    <i data-feather="thumbs-down" class="mb-1"></i> Dislike
+                                </li>
+                                <li class="nav-item ms-4">
+                                    <i data-feather="share-2" class="mb-1"></i> Share
+                                </li>
+                                <li class="nav-item ms-4">
+                                    <i data-feather="scissors" class="mb-1"></i> Clip
+                                </li>
+                                <li class="nav-item ms-4">
+                                    <i data-feather="plus-square" class="mb-1"></i> Save
+                                </li>
+                            </ul>
+                        </div>
+                        <div class="creator-details mt-3">
+                            <div class="container">
+                                <div class="row justify-content-between">
+                                    <div class="d-flex justify-content-between col-md-12 col-sm-12 col-lg-12 col-xl-5 col-xxl-5 border border-secondary one-of-two-box p-1">
+                                        <div class="creator-left col-6 d-flex justify-content-start">
+                                            <img src="img/trey-songz.jpg" alt="Trey Songz" class="rounded-circle m-auto" style="width: 40px; height: 40px;">
+                                            <div class="m-auto creator-name-subs">
+                                                <span class="">Anikriz Media</span> <br>
+                                                <span style="font-size: 10px;" class="">500 subscibers</span>
+                                            </div>
+                                        </div>
+                                        <div class="creator-right col-6">
+                                            <div class="creator-inner">
+                                                <button type="button" class="btn btn-danger">SUBSCRIBE</button>
+                                                <i data-feather="bell"></i>
+                                            </div>
+                                        </div>
+                                    </div>
+                                    <div class="d-flex justify-content-between col-md-12 col-sm-12 col-lg-12 border border-secondary one-of-two-box p-1">
+                                        <div class="col-4 comment-and-likes m-auto">
+                                            <span class="">Comments</span> <br>
+                                            <span style="font-size: 10px;" class="">1.1K</span>
+                                        </div>
+                                        <div class="col-8 d-flex justify-content-start">
+                                            <img src="img/trey-songz.jpg" alt="Trey Songz" class="rounded-circle mt-2" style="width: 25px; height: 25px;">
```

Comment:
Please do not use inline styles. Keep your style definitions in the external CSS file you created.